modules/user_accounts/test/TestPlan.md

```diff
@@ -59,18 +59,29 @@ When creating or editing a user: (subtest: `edit_user`)
     changes. 
 25. When editing an existing user: Clicking on the 'User Account' breadcrumb takes you to the User Account page
     without saving any changes to the user profile.
+    
+### Testing permissions:
+*Log in as a simple user (non-admin) and ensure the following. Define `editee` as the user being edited and `editor` as the logged in user making the changes.*
+
+26. Editor can only see and manipulate permissions for editee if and only if editor has the permissions in question
+1. When page is saved with any set of selected permissions, on reload all selected permissions are kept and all unselected permissions are removed.
+1. If editee has permissions that editor does not have, said permissions are not impacted by manipulating the editees's visible permissions (requires admin user login in a side window to be able to view all permissions for editee as editor will only be able to view permissions they own)
```

Comment:
remove `s` from second iteration of word `editee` in the sentence: ..... impacted by manipulating the *editee's* visible permissions ...